docs/maintainer/infrastructure.md

```diff
@@ -469,6 +469,159 @@ uploaded from the new feedstock, by design.
 Once this is done correctly and the package is uploaded,
 you can then request the conda-forge core devs to archive the old feedstock.
 
+## How package building interacts with infrastructure
+
+Packages in conda-forge are always built through CI.
+However, when a new package enters conda-forge for the first time, it does so via a pull request in the [`staged-recipes` repository](#staged-recipes), whereas every new build of the package after that is built in its repository, the so-called feedstock.
+Both places use slightly different CI setups and interact with the infrastructure accordingly.
+Hence, we first describe the interaction at the start of a new package and then for existing packages in their respective feedstocks.
+
+### Initial submission to staged-recipes
+
+The `conda-forge/staged-recipes` repository uses several pieces of infrastructure.
+
+On pull requests:
+
+- Package building pipelines. These are slightly different than the ones running in feedstocks (they are not automatically generated by `conda-smithy`, but they do use the same underlying components).
+- The linter is provided by `conda-smithy recipe-lint`, run by `@conda-forge-linter`.
+- Auto-labeling logic, run by Github Actions workflows.
+
+On pushes to `main`:
+
+- Create and initialize the new feedstocks.
```

Comment:
This is delegated to an external repo: [admin-requests](https://github.com/conda-forge/admin-requests/blob/main/.github/workflows/create_feedstocks.yml).